src/capi/azext_capi/_completers.py

```diff
@@ -75,5 +75,5 @@ def _get_location_from_resource_group(cli_ctx, resource_group_name):
         # Print a warning if the user hit [TAB] but the `--resource-group` argument was incorrect.
         # For example: "Warning: Resource group 'bogus' could not be found."
         from argcomplete import warn
-        warn('Warning: {}'.format(err.message))
+        warn(f'Warning: {err.message}')
```

Comment:
Python [f-strings](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals) are cool, and since we require v3.6 or later, we can use them.